docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     restart: always
     environment:
       API_URL: http://web:3001
+      LEGACY_API_URL: http://web:3001
```

Comment:
This is to make it more clear what this API is, as we are integrating APIs for new services.

Reply:
Noted in the PR description, I decided not to switch the `API_URL` references for now, because that will require all deploy scripts to be updated.  I will do that in a separate PR, to avoid adding complexity here.